src/CliptoExchange.sol

```diff
@@ -82,67 +61,82 @@ contract CliptoExchange is ReentrancyGuard {
         address requester;
         /// @dev Amount of L1 token set for the request
         uint256 amount;
-        /// @dev Whether the request is delivered
-        bool delivered;
-        /// @dev flag to indicate whether the request was refunded
-        bool refunded;
+        /// @dev Boolean indicating whether the request has been fulfilled/refunded.
+        bool fulfilled;
```

Comment:
This means we're not distinguishing on-chain between refunds & deliveries. Difficult for a backend to keep track of this, could we do an int 0 = requested, 1 = delivered, 2 = refunded?

Reply:
This should be ok, because we still differentiate between the two using events (when a refund occurs, we emit the refund event, and when a delivery occurs we emit a delivery event). Up to you tho ofc!

Reply:
I guess I can handle in frontend - just means more work there & considering time as a resource. Should be fine then - will resolve.